new-integration-tests/src/test/resources/istio/istio-cdt-http-template-service.yaml

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2020, Oracle Corporation and/or its affiliates.
+# Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+apiVersion: networking.istio.io/v1alpha3
+kind: Gateway
+metadata:
+  name: istio-cdt-http-gateway
+  namespace: NAMESPACE
+spec:
+  selector:
+    istio: ingressgateway
+  servers:
+    - hosts:
+        - '*'
+      port:
+        name: http
+        number: 80
+        protocol: HTTP
+---
+apiVersion: networking.istio.io/v1alpha3
+kind: VirtualService
+metadata:
+  name: istio-cdt-http-virtualservice
+  namespace: NAMESPACE
+spec:
+  gateways:
+    - istio-cdt-http-gateway
+  hosts:
+    - 'domain1-NAMESPACE.org'
+  http:
+    - match:
+        - uri:
+            prefix: /console
+        - uri:
+            prefix: /management
+        - uri:
+            prefix: /TxForward
+        - port: 7001
+      route:
+        - destination:
+            host: ADMIN_SERVICE
+            port:
+              number: 7001
+
+    - match:
+        - uri:
+            prefix: /TxForward
```

Comment:
This is the only difference between this template and existing istio-http-template.yaml. We can make the application prefix as replaceable.

Reply:
I also want to prefix my hostname with the domain name and namespace - not to have conflicts later - if we have 2 domains in the same namespace.

Reply:
Also, I have to deploy my app to the AS as well.

Reply:
Keeping it separate for now.

Reply:
Keeping it separate for now.